core-it-support/core-it-plugins/maven-it-plugin-class-loader/maven-it-plugin-class-loader/pom.xml

```diff
@@ -44,6 +44,10 @@ under the License.
   </description>
   <inceptionYear>2008</inceptionYear>
 
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
```

Comment:
This project is not using the parent (since it apparently cannot as described in the comment), so some of the config needs to be duplicated.